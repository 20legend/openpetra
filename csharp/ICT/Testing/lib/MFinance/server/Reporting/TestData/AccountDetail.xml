<?xml version="1.0"?>
<Parameters>
  <Parameter id="xmlfiles" value="eString:Finance/accountdetail.xml,Finance/accountdetailcommon.xml,Finance/finance.xml,common.xml" />
  <Parameter id="currentReport" value="eString:Account Detail" />
  <Parameter id="MaxDisplayColumns" value="eInteger:6" />
  <Parameter id="param_diff_period_i" value="eInteger:0" />
  <Parameter id="param_account_hierarchy_c" value="eString:STANDARD" />
  <Parameter id="param_currency" value="eString:Base" />
  <Parameter id="param_period" value="eBoolean:true" />
  <Parameter id="param_date_checked" value="eBoolean:false" />
  <Parameter id="param_start_period_i" value="eInteger:1" />
  <Parameter id="param_end_period_i" value="eInteger:1" />
  <Parameter id="param_year_i" value="eInteger:0" />
  <Parameter id="param_sortby" value="eString:Account" />
  <Parameter id="param_account_codes" value="eString:*NOTUSED*" />
  <Parameter id="param_account_code_start" value="eString:&quot;0100&quot;" />
  <Parameter id="param_account_code_end" value="eInteger:9800" />
  <Parameter id="param_rgrAccounts" value="eString:AccountRange" />
  <Parameter id="param_cost_centre_codes" value="eString:*NOTUSED*" />
  <Parameter id="param_cost_centre_code_start" value="eString:&quot;0400&quot;" />
  <Parameter id="param_cost_centre_code_end" value="eInteger:9500" />
  <Parameter id="param_rgrCostCentres" value="eString:CostCentreRange" />
  <Parameter id="param_depth" value="eString:standard" />
  <Parameter id="ColumnPosition" column="0" value="eDecimal:4618441417868443648" />
  <Parameter id="ColumnPositionIndented" column="0" value="eDecimal:0" />
  <Parameter id="ColumnWidth" column="0" value="eDecimal:4611686018427387904" />
  <Parameter id="ColumnPosition" column="1" value="eDecimal:4620693217682128896" />
  <Parameter id="ColumnPositionIndented" column="1" value="eDecimal:0" />
  <Parameter id="ColumnWidth" column="1" value="eDecimal:4613937818241073152" />
  <Parameter id="ColumnPosition" column="2" value="eDecimal:4622382067542392832" />
  <Parameter id="ColumnPositionIndented" column="2" value="eDecimal:0" />
  <Parameter id="ColumnWidth" column="2" value="eDecimal:4607182418800017408" />
  <Parameter id="ColumnPosition" column="3" value="eDecimal:4622945017495814144" />
  <Parameter id="ColumnPositionIndented" column="3" value="eDecimal:0" />
  <Parameter id="ColumnWidth" column="3" value="eDecimal:4619567317775286272" />
  <Parameter id="ColumnPosition" column="4" value="eDecimal:4626041242239631360" />
  <Parameter id="ColumnPositionIndented" column="4" value="eDecimal:0" />
  <Parameter id="ColumnWidth" column="4" value="eDecimal:4613937818241073152" />
  <Parameter id="ColumnPosition" column="5" value="eDecimal:4626885667169763328" />
  <Parameter id="ColumnPositionIndented" column="5" value="eDecimal:0" />
  <Parameter id="ColumnWidth" column="5" value="eDecimal:4613937818241073152" />
  <Parameter id="param_calculation" column="0" value="eString:Debit" />
  <Parameter id="param_calculation" column="1" value="eString:Credit" />
  <Parameter id="param_calculation" column="3" value="eString:Transaction Narrative" />
  <Parameter id="param_calculation" column="4" value="eString:Start Balance" />
  <Parameter id="param_calculation" column="5" value="eString:End Balance" />
  <Parameter id="param_ledger_number_i" value="43" />
  <Parameter id="param_with_analysis_attributes" value="eBoolean:false" />
  <Parameter id="param_quarter" value="eString:false" />
  <Parameter id="param_daterange" value="eString:false" />
</Parameters>