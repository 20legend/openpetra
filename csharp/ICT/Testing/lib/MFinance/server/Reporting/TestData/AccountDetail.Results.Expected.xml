id,CostCentre/Account,Left2,Debits,Credits,,Start Balance,End Balance
0100/{ledgernumber}00/Credit,31/01/{ThisYear},Credit,,-100.00,,Gift Example,,
0100/{ledgernumber}00/Detail,Sub Total:,,0.00,-100.00,,,0.00,100.00
0100/{ledgernumber}00/Net Balance,Net Balance:,,,-100.00,,,,
"0100",Account Total:,,0.00,-100.00,,,0.00,100.00
0100/Net Balance,Net Balance:,,,-100.00,,,,
4200/{ledgernumber}00/Debit,31/01/{ThisYear},Debit,20.00,,,Expense Example,,
4200/{ledgernumber}00/Detail,Sub Total:,,20.00,0.00,,,0.00,-20.00
4200/{ledgernumber}00/Net Balance,Net Balance:,,20.00,,,,,
4200,Account Total:,,20.00,0.00,,,0.00,-20.00
4200/Net Balance,Net Balance:,,20.00,,,,,
6000/{ledgernumber}00/Debit,31/01/{ThisYear},Debit,100.00,,,Gift Example,,
6000/{ledgernumber}00/Credit,31/01/{ThisYear},Credit,,-20.00,,Expense Example,,
6000/{ledgernumber}00/Detail,Sub Total:,,100.00,-20.00,,,0.00,-80.00
6000/{ledgernumber}00/Net Balance,Net Balance:,,80.00,,,,,
6000,Account Total:,,100.00,-20.00,,,0.00,-80.00
6000/Net Balance,Net Balance:,,80.00,,,,,
6200/{ledgernumber}00/Debit,31/01/{ThisYear},Debit,40.00,,,Start Balance,,
6200/{ledgernumber}00/Detail,Sub Total:,,40.00,0.00,,,0.00,-40.00
6200/{ledgernumber}00/Net Balance,Net Balance:,,40.00,,,,,
6200,Account Total:,,40.00,0.00,,,0.00,-40.00
6200/Net Balance,Net Balance:,,40.00,,,,,
9700/{ledgernumber}00/Credit,31/01/{ThisYear},Credit,,-40.00,,Start Balance,,
9700/{ledgernumber}00/Detail,Sub Total:,,0.00,-40.00,,,0.00,40.00
9700/{ledgernumber}00/Net Balance,Net Balance:,,,-40.00,,,,
9700,Account Total:,,0.00,-40.00,,,0.00,40.00
9700/Net Balance,Net Balance:,,,-40.00,,,,
GrandTotal,Grand Total:,,160.00,-160.00,,,0.00,0.00
GrandTotalNetBalance,Net Balance:,,,0.00,,,,
